container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -163,7 +163,7 @@ pub type Executive = frame_executive::Executive<
     Block,
     frame_system::ChainContext<Runtime>,
     Runtime,
-    pallet_maintenance_mode::ExecutiveHooks<Runtime>,
+    AllPalletsWithSystem,
```

Comment:
Why this? this seems like a big change

Reply:
This is basically saying that maintenance mode is no-longer in charge of orchestrating hooks. Is this what we want?